semgrep/tests/e2e/targets/spacegrep/markdown.md

```diff
@@ -89,7 +89,7 @@ Once installed, Semgrep can be run with single rule patterns or entire rule sets
 $ semgrep -e '$X == $X' --lang=py path/to/src
 
 # Run a ruleset with rules for many languages
-$ semgrep --config=p/r2c-CI path/to/src
+$ semgrep --config=https://semgrep.dev/p/r2c-CI path/to/src
```

Comment:
wat ?

Reply:
i modified a target of tests instead of the actual readme.md. Reverted the changes to markdown.md